dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/MailMessageCreator.java

```diff
@@ -370,7 +370,7 @@ private String datasetVersionMessage(EmailNotificationDto notificationDto, Datas
                                                                                       getDatasetLink(version.getDataset()),
                                                                                       version.getDataset().getOwner().getDisplayName(),
                                                                                       getDataverseLink(version.getDataset().getOwner()),
-                                                                                      systemConfig.getGuidesBaseUrl(),
+                                                                                      systemConfig.getGuidesBaseUrl(session.getLocale()),
```

Comment:
Also it would be good idea to extract `notificationDto.getNotificationReceiver().getNotificationsLanguage()` to some variable - for example `notificationEmailLanguage` in methods.
And get rid of `private DataverseSession session;` in this class.